.github/workflows/build_and_test.yml

```diff
@@ -107,6 +107,30 @@ jobs:
     - name: Clippy
       run: cargo clippy --all -- -D warnings
 
+  rust_ios:
+    name: Build Rust for iOS
+
+    runs-on: macos-latest
+
+    needs: changes
+
+    if: ${{ needs.changes.outputs.rust == 'true' }}
```

Comment:
This can be `needs.changes.outputs.swift` to avoid changes that aren't part of libsignal-ffi.

Reply:
OTOH with that, we rebuild even on changes that only touch Swift code. I added a new filter covering just the Rust code that is relevant to iOS